indico/queries/datasets.py

```diff
@@ -306,3 +307,212 @@ def __init__(self, id):
 
     def process_response(self, response):
         return super().process_response(response)["deleteDataset"]["success"]
+
+
+#########################
+## Dataset Pipeline v2 ##
+#########################
+
+
+class CreateDataset_v2(GraphQLRequest):
```

Comment:
mmmm, I don't like this naming scheme... I think we should update the version of the client and make sure people know it only works with the new platform version.

Reply:
OK. I'll have a `CreateDatasetPipeline` that will have similar behavior to current `CreateDataset` request chain for use in existing tests that use dataset as as fixture and for clients that want to do it in one operation.

Reply:
EDIT: after subsequent conversation, we discussed keeping `CreateDataset` with same API and additionally having `CreateEmptyDataset`